src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -446,4 +458,31 @@ const StyledIcon = styled(IconStack)`
   vertical-align: middle;
 `;
 
+type TopNIndicatorProps = {
+  count: number;
+  index: number;
+};
+
+const TopNIndicator = styled('div')<TopNIndicatorProps>`
+  position: absolute;
+  left: -1px;
+  width: 9px;
+  height: 15px;
```

Comment:
From your screenshot the indicator wasn't vertically centered with the content in the row. We could use a top value to bump it down and center the indicator with the text and icon.

Reply:
Since it has `position: absolute` setting `top: x` is relative to the entire table rather than the row. Instead, I added a `margin-top: x` to offset the element in order to center it.

Reply:
Updated screenshot showing them centered